core/src/main/java/com/sequenceiq/cloudbreak/init/clustertemplate/DefaultClusterTemplateCache.java

```diff
@@ -110,7 +116,11 @@ private void loadByClasspathPath(Collection<String> names) {
                 .forEach(clusterTemplateName -> {
                     try {
                         String templateAsString = readFileFromClasspath(clusterTemplateName);
-                        convertToClusterTemplate(templateAsString);
+                        convertToClusterTemplate(
+                                clusterTemplateName,
+                                templateAsString,
+                                preferencesService.enabledPlatforms(),
+                                preferencesService.enabledGovPlatforms());
```

Comment:
you do not have to pass these 2 parameters but rather inline them in the `doUseIt` method